.github/workflows/publish_charm.yaml

```diff
@@ -72,13 +72,13 @@ jobs:
           TREE_SHA=$(gh api \
             -H "Accept: application/vnd.github+json" \
             -H "X-GitHub-Api-Version: 2022-11-28" \
-            /repos/arturo-seijas/indico-operator/commits/${GITHUB_SHA} \
+            /repos/${{ github.repository }}/commits/${GITHUB_SHA} \
              --jq '.commit.tree.sha')
           RUN_ID=$(gh api \
             --method GET \
             -H "Accept: application/vnd.github+json" \
             -H "X-GitHub-Api-Version: 2022-11-28" \
-            /repos/arturo-seijas/indico-operator/actions/runs \
+            /repos/${{ github.repository }}/actions/runs \
             -f path=.github/workflows/integration_test.yaml \
             -f tree_id=c5dafc35af124b2c75460388fa537d1738c0db76 \
```

Comment:
TREE_SHA?

Reply:
It's `tree_id`, but it's a weird working indeed

Reply:
I mean: "-f tree_id=${{TREE_SHA}}" ?
instead of fixed value "-f tree_id=c5dafc35af124b2c75460388fa537d1738c0db76"

Reply:
Oh!

Reply:
Fixed. Thanks